test/functional/pdo_sqlsrv/pdo_ae_azure_key_vault_keywords.phpt

```diff
@@ -4,9 +4,7 @@ Test connection keywords for Azure Key Vault for Always Encrypted.
 <?php require('skipif_mid-refactor.inc'); ?>
 --FILE--
 <?php
-require_once("MsCommon_mid-refactor.inc");
-require_once("MsSetup.inc");
-require_once('values.php');
+require_once('pdo_ae_azure_key_vault_common.php');
 
```

Comment:
Do you drop the your test table(s) in this test and the sqlsrv equivalent one?

Reply:
Yes, at the end of insertDataAndVerify() in the new common files.

Reply:
hmmm... I don't see this insertDataAndVerify() anywhere... not sure how I miss it?

Reply:
It's in sqlsrv_ae_azure_key_vault_common.php and pdo_ae_azure_key_vault_common.php.